siriuspy/siriuspy/servconf/types/tb_normalized.py

```diff
@@ -0,0 +1,46 @@
+"""TB normalized configurations.
+
+Values in _template_dict are arbitrary. They are used just to compare with
+corresponding values when a new configuration is tried to be inserted in the
+servconf database.
+"""
+from copy import deepcopy as _dcopy
+
+
+def get_dict():
+    """Return a dict with ramp settings."""
+    module_name = __name__.split('.')[-1]
+    _dict = {
+        'config_type_name': module_name,
+        'value': _dcopy(_template_dict)
+    }
+
+    return _dict
+
+
+_template_dict = {
+    'pvs': [
+        ['TB-Fam:PS-B:Energy-SP', 0.0, 0.0],  # [GeV]
+        ['TB-01:PS-QD1:KL-SP', 0.0, 0.0],  # [1/m]
```

Comment:
I think the discipline may be wrong here: 'TB-Fam:**MA**-B:Energy-SP' instead of 'TB-Fam:PS-B:Energy-SP'

Reply:
Thank, @anacso17 . I pushed a fix to this issue.